nitro-protocol/hardhat-deploy-fvm/00-deploy.ts

```diff
@@ -0,0 +1,31 @@
+import 'hardhat-deploy';
+import 'hardhat-deploy-ethers';
+
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {BigNumber} from 'ethers';
+
+module.exports = async (hre: HardhatRuntimeEnvironment) => {
+  const {deployments, getNamedAccounts, getChainId} = hre;
+  const {deploy} = deployments;
+  const {deployer} = await getNamedAccounts();
+
+  try {
+    console.log('Working on chain id #', await getChainId());
+
+    await deploy('NitroAdjudicator', {
+      from: deployer,
+      args: [],
+      // since it's difficult to estimate the gas limit before f4 address is launched, it's safer to manually set
+      // a large gasLimit. This should be addressed in the following releases.
+      gasLimit: 1000000000, // BlockGasLimit / 10
```

Comment:
We should probably look at getting our existing deploy script to work with wallaby, but I think that can be done outside the scope of this PR.

Reply:
This was copied over from the old fevm deploy script. I've removed it.